### ISSUE TITLE
Reduce explosion time of buildings when control-K'd

### ISSUE THREAD

**Issue by IndexLibrorumProhibitorum** [FAForever/fa#4772]
**Describe the feature**
The current explosion takes quite a bit of time. Please reduce the duration. 

**Additional context**
The matter came up in a recent discussion on our group's discord, several people expressed their displeasure with the current time it takes. Alternatively, allow buildorders to be placed on a structure that has yet to finish exploding.

**Comment by Garanas** [comment]
> Alternatively, allow buildorders to be placed on a structure that has yet to finish exploding.

This is not possible.

But I'm in favor of reducing the explosion time in general.

**Comment by Garanas** [comment]
I personally think it would be interesting to have more unique death threads for various structures. That way we can drop the 'generic explosion' that applies to almost every unit except for experimental units.

**Comment by MrRowey** [comment]
> I personally think it would be interesting to have more unique death threads for various structures. That way we can drop the 'generic explosion' that applies to almost every unit except for experimental units.

So something like RK Explosions?

**Comment by Garanas** [comment]
Yes, but then less excessive. I may do a small setup this month, and then we can investigate whether that is the direction we'd like to take.

**Comment by lL1l1** [comment]
@IndexLibrorumProhibitorum does this mostly pertain to Seraphim structures, or something else specifically? Because Seraphim structures are being addressed here: https://github.com/FAForever/fa/issues/6613

If not about only Seraphim structures (their death times are simply being normalized), then it'd be nice to have some reasoning and examples to be able to discuss this from a gameplay PoV.
